pkg/operator/ceph/cluster/controller.go

```diff
@@ -401,12 +401,25 @@ func (c *ClusterController) reconcileNodeMaintenance(updatedNode *v1.Node) {
 					break
 				}
 			}
+			osdDump, err := client.GetOSDDump(c.context, cluster.Info.Name)
```

Comment:
If `GetOSDDump()` is called inside the set/unset methods, why is it needed here? If it's for the logging, I wonder if we could move the logging inside those methods, or else remove the `GetOSDDump()` call from those methods since it is redundant.

Reply:
ugh, could've sworn I removed it from set/unset. Fixing!